app/views/common/_login.html.erb

```diff
@@ -1,2 +1,3 @@
-<div id='login-page'></div>
-<%= mount_react_component('LoginPage', '#login-page', props.to_json, flatten_data: true)%>
+<div id='login-page'>
```

Comment:
`react_component` doesn't need a div anymore for mounting, how about moving the `div` into the component? the correlated css rules are already under wepback

Reply:
Sure